src/actions/guides/deploying/dokku.cr

```diff
@@ -0,0 +1,99 @@
+class Guides::Deploying::Dokku < GuideAction
```

Comment:
Could you run `crystal tool format src` to fix formatting? It looks like there is a formatting issue somewhere in here according to CI

Reply:
Done :)